lib/engine/game/base.rb

```diff
@@ -659,6 +659,10 @@ def redeemable_shares(_entity)
         []
       end
 
+      def allow_percentage_on_buy(_share)
```

Comment:
what does this mean?

Reply:
1860 needs a way to suppress printing out "20%" on the buy from market button for the president's share. No other game needs that. This was the least intrusive way I could think of doing that.